js/components/Vizr.vue

```diff
@@ -17,7 +20,8 @@
                :metadata="metadata"
                :chart-def="chart"
                @delete-chart="deleteChart"
```

Comment:
Previously we needed a special case for toggling the legend, so that we didn't fetch the data again. Because of Vue's reactivity, there is no longer a special check. The chart component simply detects that the chart definition hasn't changed and doesn't re-fetch the data.